src/compressor.h

```diff
@@ -19,7 +19,17 @@ bool CompressScript(const CScript& script, std::vector<unsigned char> &out);
 unsigned int GetSpecialScriptSize(unsigned int nSize);
 bool DecompressScript(CScript& script, unsigned int nSize, const std::vector<unsigned char> &out);
 
+/**
+ * Compress amount.
+ *
+ * Please note that nAmount is of type uint64_t and thus cannot be negative.
```

Comment:
```suggestion
 * nAmount is of type uint64_t and thus cannot be negative.
```

"Please note that" is an empty phrase and can be removed.

Reply:
Good point! Thanks! Now fixed.